src/us/kbase/narrativejobservice/sdkjobs/SDKLocalMethodRunner.java

```diff
@@ -367,7 +367,7 @@ public boolean isJobCanceled() {
                         requestedRelease);
                 final CallbackServerConfig cbcfg = 
                         new CallbackServerConfigBuilder(config, callbackUrl,
-                                jobDir.toPath(), log).build();
+                                jobDir.toPath(), refDataDir.toPath(), log).build();
```

Comment:
I'm not sure if this will work.  I think this will wind up passing the refDataDir for the "parent" SDK module, not the refDataDir for the subjob.  I think you will need to replicate the logic here...

https://github.com/kbase/njs_wrapper/blob/develop/src/us/kbase/narrativejobservice/sdkjobs/SDKLocalMethodRunner.java#L216-L225

Down in the NJSSubsequentCallRunner.

Reply:
Made this update -- see comments in PR